cmd/soroban-rpc/main.go

```diff
@@ -296,12 +294,10 @@ func main() {
 	cmd.AddCommand(versionCmd)
 
 	if err := configOpts.Init(cmd); err != nil {
-		supportlog.New().WithError(err).Fatal("could not parse config options")
-		os.Exit(-1)
+		fmt.Fprintf(os.Stderr, "could not parse config options: %v\n")
 	}
 
 	if err := cmd.Execute(); err != nil {
-		supportlog.New().WithError(err).Fatal("could not run")
-		os.Exit(-1)
+		fmt.Fprintf(os.Stderr, "could not run: %v\n", err)
```

Comment:
If we have an error which makes us exit the process, we should probably exit with a non-zero. This is particulary important for automation scripts.

Reply:
Yep, it was an oversight. Thanks